### PR TITLE
get tests endpoint added

### PR DESCRIPTION
**Pull request by katarinarosiak**
Co-authored-by: Katarina Rosiak <katarinarosiak@gmail.com>
Co-authored-by: Tim Dronkers <tdronkers@gmail.com>
Co-authored-by: Scott Graham <scttgrhm7+public@gmail.com>
Co-authored-by: Miles Abbason <miles.abbason@gmail.com>

Done:
- added one endpoints: 
1. GET '/tests': getting all scheduled tests (those will be displayed on the Home page) 
2. added cors policy

### CODE REVIEW

File: routes/api.js [hunk 1]
```diff
@@ -4,5 +4,7 @@ const testsController = require('../controllers/testsController');
 const router = express.Router();
 
 router.post('/tests', testsController.createTest);
+router.get('/tests', testsController.getScheduledTests);
+router.get('/tests/results', testsController.getTestsResults);
```
Comment (scogra17) [inline routes/api.js]: I'm wondering if we should just collapse this into a single endpoint `/tests` that gets all tests and the last test result. What do you think?

**Review by scogra17 — CHANGES_REQUESTED** [review]
Looks great overall! I added a comment about collapsing the two endpoints into one; maybe we can discuss during our standup?

**Review by tdronkers04 — APPROVED** [review]
Great job Katarina!

I'm wondering if we should re-think our function naming conventions (my own included). 

For example, `getTestsResults` calls a function `getResults` (the actual db query). `createTest` ultimately calls a function called `addNewTest` (the db query). 

Would it make sense to append the db-query functions with something like `getResultsDB`

**Review by scogra17 — APPROVED** [review]
Looks great!

**Pull request merged**